.github/workflows/c2a.yml

```
@@ -10,4 +10,4 @@ on:
 jobs:
   c2a:
     name: C2A
-    uses: arkedge/workflows-c2a/.github/workflows/default.yml@v4.5.3
+    uses: arkedge/workflows-c2a/.github/workflows/default.yml@v5.0.0
```

このバージョンから c2a-core v4 をデフォルトでは想定することになったため，`default-v3.yml` に移行する必要があります．
```suggestion
    uses: arkedge/workflows-c2a/.github/workflows/default-v3.yml@v5.0.0
```